clients/venice-admin-tool/build.gradle

```diff
@@ -21,6 +22,8 @@ dependencies {
     // Helix use zk 3.6.9, which introduce netty 3.10 and will fail our test.
     exclude module: 'zookeeper'
   }
+
+  testImplementation project(path: ':internal:venice-test-common', configuration: 'integrationTestUtils')
```

Comment:
We do not want to set up integrationTest clusters for unit tests. Can we either add mock for the unit tests, or move this test to an integration test package?

Reply:
If you had to do this to pass the coverage check, let's reduce the threshold if it is not straightforward to write tests for `AdminTool`

Reply:
yeah, as mentioned in the PR description, it is hard to write meaningful mock test with the current structure.